### DISCUSSION TITLE
deleted file still tracking by git

### DISCUSSION THREAD

**Discussion by jacobscript** [community/community#106695] [New to GitHub]
The big  .rar file was deleted from the project folder outside vs code. Now I am trying to push to GitHub and get an error message this (deleted) file exceeds the size limit. Already did the PC restart and 'git add -u'. What else can I do?

**Comment by rafidRD** [comment]
You have to fetch the git... it's like refresh button

**Reply by jacobscript** [reply]
Thanks, but the problem is still here.

**Comment by vix-4800** [comment]
Hey!

Git has a cache that sometimes needs to be updated manually.
When a file is added to the staging area (index) using `git add` and commited (`git commit`), git stores a copy of that file in its cache. This copy is used to compare changes and efficiently track modifications.

You can view files in git's cache by running
`git ls-files --cached`
If you see your file there, you might want to try cleaning the git cache to ensure that it reflects the current state of your repository
`git rm -r --cached`
or
`git rm <filename> --cached` to delete a specific file from cache

Another potential reason is that the file was previously added to the staging area using `git add`.
If you deleted the file without updating the git index, it remains in the staged changes. Git, being instructed to track the file, attempts to push it.

To check for files in the staging area, use
`git status`
and look for the line 'Changes to be committed:'.
If you find the deleted file listed there, you can remove it from the index by running
`git reset <filename>`

Hope this helps!😄
